backend/app/controllers/spree/admin/reports_controller.rb

```diff
@@ -34,19 +34,34 @@ def index
       end
 
       def show
-        report_class = "Spree::Report::#{params[:id].camelize}".constantize
-        report = report_class.new(params)
-        respond_to do |format|
-          format.html do
-            report.content.each do |key, value|
-              instance_variable_set("@#{key}", value)
+        report_class = "Spree::Report::#{params[:id].camelize}".safe_constantize
+        if report_class.present?
```

Comment:
I've always found `.present?` a bit more readable, but I'm happy to match code style elsewhere in Solidus.